src/Toml/BiMap.hs

```diff
@@ -145,6 +157,31 @@ _ShowString = BiMap (Just . show) readMaybe
 _Show :: (Show a, Read a) => BiMap a AnyValue
 _Show = _ShowString >>> _String
 
+_UnsignedInteger :: (Integral n) => BiMap n Integer
+_UnsignedInteger = prism fromIntegral maybeInteger
+  where maybeInteger n
+          | n < 0     = Nothing
+          | otherwise = Just (toInteger n)
+
+_Natural :: BiMap Natural AnyValue
+_Natural = _UnsignedInteger >>> _Integer
+
+_Word :: BiMap Word AnyValue
+_Word = _UnsignedInteger >>> _Integer
+
+_RealReal ::  (Real a, Real b, Fractional a, Fractional b) => BiMap a b
+_RealReal = iso realToFrac realToFrac
```

Comment:
I feel like `_RealReal` is not going to be used outside `_Float`...

Reply:
That's very likely. I removed it from the list and only use it in `_Float`.